### ISSUE TITLE
Builtin Methods which Maps to JavaScript Methods

### ISSUE THREAD

**Issue by Maxwell-Alexius** [Maxwell-Alexius/Wyrd-Lang#46]
**What?**
Create builtin method for basic operations.

**Why?**
Currently there is no way of operating values with its type's methods. For instance, we can't do anything like calling `toStr` method for `Str` type data.

**How?**
For instance, the `Str.toStr(self: Str): Str` is a method which reverses the character of the string.

Sample Wyrd Code
```
# Calls method using Str.reverse
Str.toStr("Hello world")

# or performs on the "Hello world" string value data
"Hello world".toStr()
```

Compiled Result
```js
("Hello world").toString();
("Hello world").toString();
```

**Comment by Maxwell-Alexius** [comment]
Implementing via PR #47

**Comment by Maxwell-Alexius** [comment]
Currently only implemented the `"Hello world".toStr()` version

**Comment by Maxwell-Alexius** [comment]
This has been solved from PR #49
